lotto/src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,51 @@
+package lotto.view
+
+class InputView {
+
+    companion object {
+
+        fun inputBuyMoney(): Long {
+            println("구입금액을 입력해 주세요.")
+            val input = readln()
+
+            runCatching {
+                input.toLong()
+            }.onSuccess {
+                return input.toLong()
+            }.onFailure {
+                println("숫자를 입력해주세요")
+            }
+            return inputBuyMoney()
+        }
+
+        fun inputWinningLotto(): List<Int> {
+            println("지난 주 당첨 번호를 입력해 주세요.")
+            val input = readln()
+
+            kotlin.runCatching {
```

Comment:
```suggestion
            runCatching {
```
runCatching에 대해 배워갑니다 :) 다만 kotlin은 빼도 될 것 같아요!